### PR TITLE
added menu to header, fixes #5

### LINKED ISSUE #5: Should be a menu in the top header
There should be a `nav` list of with the following links:

- Home: `/`
- About Us: `/#about-us`
- Residential: `/#residential`
- Contact Us: `/#contact-us`
- Portfolio: `/portfolio`

When you click on any of these links they should navigate (either scroll or change pages) to the link's target (ie. clicking on Contact Us should scroll the page to the Contact Us page)

### PR DESCRIPTION
**Pull request by njosieb**
Added menu to header of page. @cvharris

### CODE REVIEW

File: src/_layouts/header.njk [hunk 1]
```diff
@@ -16,5 +16,11 @@
     <div id="menu-reveal" class="menu-button-container right">
       Menu&nbsp;<i class="fa fa-bars"></i>
     </div>
+    <div class="menu">
+        <a title="Home" href="/" class="menu-item"></a>
+        <a title="About Us" href="/#about-us"></a>
+        <a title="Residential" href="/#residential"></a>
+        <a title="Contact Us" href="/#contact-us"></a>
+    </div>
```
Comment (cvharris) [inline src/_layouts/header.njk]: We could actually make use of Nunjuck's built-in [for-loop](https://mozilla.github.io/nunjucks/templating.html#for) to make this a lot more simple, so we can make use of everything in the `menu.yml`

File: src/_layouts/header.njk [hunk 2]
```diff
@@ -16,5 +16,11 @@
     <div id="menu-reveal" class="menu-button-container right">
       Menu&nbsp;<i class="fa fa-bars"></i>
     </div>
+    <div class="menu">
+        <a title="Home" href="/" class="menu-item"></a>
+        <a title="About Us" href="/#about-us"></a>
+        <a title="Residential" href="/#residential"></a>
```
Comment (cvharris) [inline src/_layouts/header.njk]: this should ultimately look like `<a title="{$ link.description $}" href="{$ link.route $}" class="menu-item">{$ link.label $}</a>`

**Review by cvharris — CHANGES_REQUESTED** [review]

**Comment by njosieb** [comment]
@cvharris  Please look at adjustments I made to make sure it works like it should.

**Review by cvharris — APPROVED** [review]
Awesome! 👍 Merge this!

**Pull request merged**